types/eslint-scope/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for eslint-scope 3.7
+// Project: http://github.com/eslint/eslint-scope
+// Definitions by: Toru Nagashima <https://github.com/mysticatea>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+import * as eslint from "eslint";
+import * as estree from "estree";
+
+export const version: string;
+
+export class ScopeManager implements eslint.Scope.ScopeManager {
+    scopes: Scope[];
+    globalScope: Scope;
+    acquire(node: {}, inner?: boolean): Scope | null;
+    getDeclaredVariables(node: {}): Variable[];
```

Comment:
Instead of `{}`, why not use an estree Node? (here and in other locations)

Reply:
ESLint AST could not assign to `@types/estree` AST types, so I'd like to keep relax here.
(though I have ESLint AST definitions in my project local [vue-eslint-parser/src/ast/nodes.ts](https://github.com/mysticatea/vue-eslint-parser/blob/6111437a4b470fb63227099a69749a7ef2f022a4/src/ast/nodes.ts), I want to find why it cannot assign to estree types. But the type definition is recursive, so `tsc` doesn't show meaningful error :cry:)

Reply:
`eslint-scope` uses those parameters as only the key of `Map`, so I think this relaxing does not trigger runtime errors.